eng/optimizationData.targets

```diff
@@ -0,0 +1,28 @@
+﻿<Project>
+
+  <ItemGroup>
+    <IBCPackage Include="$(WindowsCoreFxOptimizationDataPackageId)" Version="$(optimizationwindows_ntx64IBCCoreFxPackageVersion)" />
+    <IBCPackage Include="$(LinuxCoreFxOptimizationDataPackageId)" Version="$(optimizationwindows_ntx64IBCCoreFxPackageVersion)" />
+    <PackageReference Include="@(IBCPackage)" PrivateAssets="all" IsImplicitlyDefined="true" />
+  </ItemGroup>
+  
+  <!-- IBC data packages don't follow NuGet conventions.  -->
+  <Target Name="GetIBCData"
+          Inputs="%(IBCPackage.Identity)"
+          Outputs="$(IbcOptimizationDataDir)"
+          AfterTargets="Restore">
```

Comment:
This cannot be AfterTargets="Restore" because it relies on properties that NuGet writes to the props file, so you must re-evaluate the project after restore happens.  Alternatively you can just construct the path to the package like we do elsewhere rather than rely on the generated property.

Reply:
Not sure which properties you mean here? I tested this and it works fine.